server/src/main/java/org/opentosca/toscana/api/TransformationController.java

```diff
@@ -778,18 +778,16 @@ This Mapping is used to post the inputs (properties) to the server. this will se
                 successes.put(entry.getKey(), false);
             }
         }
-        //Return the result (with code 200 if all inputs were valid and 400 if at least 1 was invalid)
-//        PropertyInstance instance = transformation.getProperties();
-//        //TODO if other requirement types get used, this needs a change!
-//        //Change state of the transformation to show the user that all required properties have been set
-//        if(instance.requiredPropertiesSet(RequirementType.TRANSFORMATION)) {
-//            //TODO Maybe a different method to change the state is needed!
-//            transformation.setState(TransformationState.READY);
-//        }
         if (!somethingFailed) {
             return ResponseEntity.ok().build();
         } else {
-            return ResponseEntity.status(HttpStatus.NOT_ACCEPTABLE).body(new SetPropertiesResponse(successes));
+            SetPropertiesResponse response = new SetPropertiesResponse(
+                csarId,
+                platformId,
+                setPropertiesRequest.getProperties(),
```

Comment:
I think this will lead to faulty behaviour in combination with default values.

Imagine a SET properties request containing this property:
```yaml
property:
  value: null
  required: true
  default: "my-default"
  type: text
```

What currently happens: You send the same property back (including success indicator)

But given the fact that the property is required and has a default value, following behaviour will be observable:
```java
property.setValue(null);
assertEquals(property.getDefault(), property.getValue());
assertNull(property.getValueWithoutDefault);
```

The api response should also reflect this evaluation.
Not sure whether it's still appropriate to not have default values in the `PropertyWrap` class.